GVFS/GVFS.UnitTests/Mock/Common/MockPlatform.cs

```diff
@@ -170,6 +170,11 @@ public override string WorkingDirectoryBackingRootPath
                 get { return GVFSConstants.WorkingDirectoryRootName; }
             }
 
+            public override string DotGVFSRoot
+            {
+                get { return ".vfsforgit"; }

```

Comment:
Change this to `".mockvfsforgit"` to make it easier to validate that nothing has a dependency on this specific string?